src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -12,40 +12,44 @@ redirects:
-5. Optional: Set up alert notifications.
-6. Select **Create my monitor** to confirm.
-7. Wait a few minutes; then from the [Monitors index](#bookmark) check your monitor.
 
-## 2. View all monitors with the Monitors index page [#bookmark]
+   * **Bypass HEAD request (for ping monitors).** This option skips the default HEAD request and instead uses the GET verb with a ping check. GET requests will always happen if the HEAD request fails.
+   * **Redirect is Failure (for ping).** If a redirect result occurs when `Redirect is Failure` is enabled, your synthetic monitors categorize it as a failure (rather than following the redirect and checking the resulting URL).
+
+4. Select the locations where you want your monitor to run. It is recommended to select at least three locations to avoid false positives. In other words, if at least one location returns a successful result then the endpoint must be up and triggering of an alert can be avoided.
+5. Depending on the monitor type, choose to either Save monitor, Validate, or Write script.
+6. You'll be taken to the [Summary page](#summary-page) where results will start to appear as they are received.
+
+## 2. Get a quick summary of how a monitor is running [#summary-page]
+
+TBD, screenshots, what's useful on this page?
+
+Once on the Summary page, you'll be presented with the most relevant information to assist you in determining the status of this monitor. If an alert has triggered and created an active incident, click to open the critical alert in a new tab for this monitor, or click to manage alert policies for all monitors at the top of the page.
+![](/src/images/synthetics-monitor-critical-alert.png)
+
+## 3. View all monitors on the Synthetics Monitors page [#synthetics-monitors]
 
-Continuous application performance monitoring is essential to ensure that web services are in place, working correctly, and error-free. Synthetic monitoring provides this type of assurance by performing automated tests on your web application for each selected location—noting downtime instances (“violations”) and collecting aggregate numbers, results, and detailed statistics for each page resource.
+Continuous application performance monitoring is essential to ensure that web services are in place, working correctly, and error-free. Synthetic monitoring provides this type of assurance by performing automated tests on your web application for each selected location—noting downtime instances ("violations") and collecting aggregate numbers, results, and detailed statistics for each page resource.
 
 Use the Monitors index page to get a high level view of this information, or select an individual monitor to view the [Summary](/docs/new-relic-one-monitors-index), for ping monitors, or [Overview](/docs/synthetics/new-relic-synthetics/pages/synthetics-overview-page-view-monitors-performance), for simple and scripted monitors, page and get a deeper insight into its performance over time.
```

Comment:
official name in the UI?

Reply:
@keegoid-nr What is 'this information'? Could we say, '....to get a high level view of what your monitors are reporting to New Relic'?

Reply:
I modified some language for clarity/readability. Can you go into a little more detail about the differences between the summary page and overview page? What do each provide that the other does not? Are they different areas for different monitors because monitors vary in complexity? Which monitor type corresponds to which page? This is a little vague and confusing, and by only mentioning a handful of monitors, it looks like we're giving incomplete information.

Reply:
re: the Collapsers below and re: comment above ^, it seems like there are 2 ways to view the same kind of information (your monitors). If you were recommending something to a customer, what would be your recommendation for looking at all your monitors? If there are differences, what are they? What are the benefits of viewing performance/monitors in one over the other?

Reply:
@akristen The names are confusing here. The index page is a list of all monitors and is now rightly called the `Index of synthetic monitors`. I will submit an update for this section since the current version is not accurate.

To address your questions, the index is useful quickly see which monitors are failing. The Summary page would then be used to see where it is failing, why it is failing (http codes), and other information about that one monitor of interest.